src/App.js

```diff
@@ -483,7 +483,8 @@ function App() {
 
               playNext();
             }}
-            autoPlay={autoplay}
+            // set autoPlay for taps and keyboard input
+            autoPlay="true"
```

Comment:
```suggestion
            autoPlay={true}
```

react suggests this construction